File: PROJECT.BE/PROJECT.Service/Extention/PaginationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PROJECT.Service.Extention
{
    public class PaginationModel
    {
        public int CurrentPage { get; set; }
        public int TotalPage { get; set; }
        public int ItemCount { get; set; }
        public int PageSize { get; set; }
        public string KeySearch { get; set; }   
        public object Data { get; set; }
        
    }
}

File: PROJECT.UI/src/app/models/Common/pagination.model.ts
export interface Pagination {
  CurrentPage: number;
  TotalPage: number;
  PageSize: number;
  KeyWord: string;
  TotalRecord: number;
  Data?: object;
}

export class PaginationResult {
  currentPage: number = 0;
  totalPage: number = 0;
  pageSize: number = 0;
  keyWord: string = '';
  totalRecord: number = 0;
  data?: any;
}
